allen-cell-animated/nucmorph-colorizer
# refactor: Moved tab contents to their own components

Problem
=======
Refactors the tab view in `App.tsx`.

Solution
========
- Moves the Plot and Settings tabs to their own components.
- Renames `FilterThresholdPanel.tsx` to `FilterThresholdsTab.tsx`.
- Adds styled utility components (`FlexDiv`, `FlexColumn`, `FlexRow`, `FlexRowCentered`) for boilerplate CSS styling.

Steps to Verify:
----------------
1. Open the preview link and click around on the panel view. Everything should behave as it did before.
